### PR TITLE
CAV-445: Standardise API responses - Verify endpoint to return status…

### PR DESCRIPTION
**Pull request by johno1985**
… endpoint url in the Location header

 - refactored verify response, pass headers through

### CODE REVIEW

**Comment by johno1985** [comment]
rebuild

**Comment by johno1985** [comment]
PR job is failing only because changes to verification service have not yet been merged. All tests on all repos are passing locally

**Comment by mar36tin** [comment]
also tests are failing, please have a look

**Comment by johno1985** [comment]
rebuild

**Review by mar36tin — APPROVED** [review]

File: test/uk/gov/hmrc/cipphonenumber/controllers/VerifyControllerSpec.scala [hunk 1]
```diff
@@ -41,37 +41,40 @@ class VerifyControllerSpec extends AnyWordSpec
...
 
       status(response) shouldBe BAD_REQUEST
       contentAsJson(response) shouldBe Json.parse("""{"res":"res"}""")
+      header("header-name", response) shouldBe Some("header-value")
     }
 
     "convert upstream 500 response" in {
       mockVerifyConnector.verify(Json.parse("""{"req":"req"}"""))(any[HeaderCarrier])
-        .returns(Future.successful(HttpResponse(INTERNAL_SERVER_ERROR, "")))
+        .returns(Future.successful(HttpResponse(INTERNAL_SERVER_ERROR, "", Map("header-name" -> Seq("header-value")))))
```
[the remark this answers is not in the document]
Comment (johno1985) [reply test/uk/gov/hmrc/cipphonenumber/controllers/VerifyControllerSpec.scala:71]: just a made up name for the header in the unit test

**Review by mar36tin — APPROVED** [review]

**Pull request merged**